edx-platform/learninghub-theme/lms/templates/instructor/edx_ace/enrollenrolled/email/body.html

```diff
@@ -1,95 +1,184 @@
+                                {{ site_name }}
+                            </a>
+
+                            <span style="color:#333333">
+                                &nbsp;by a member of the course staff. The course should now appear on your&nbsp;
+                            </span>
+
+
+                            <a href="{{ site_name }}" rel="noreferrer" style="color:#007fa3;text-decoration:underline" title="" target="_blank" data-saferedirecturl="{{ site_name }}">
+                                {{ site_name }}
+                            </a>
+
+                            <span style="color:#333333">
+
+                                &nbsp;dashboard.
+                                <br><br>
+
+                                To start accessing course materials, please visit
+
+                                <a href="{% with_link_tracking dashboard_url %}" style="color:#007fa3;text-decoration:underline" title="{% with_link_tracking dashboard_url %}" target="_blank" data-saferedirecturl="{% with_link_tracking dashboard_url %}">
+                                    {{ site_name }}
+                                </a>
+
+                            </span><br><br>
+
+                            <span style="color:#333333">
+                                This email was automatically sent from&nbsp;
+                            </span>
+
+                            <a href="{{ site_name }}" style="color:#007fa3;text-decoration:underline" title="{{ site_name }}" target="_blank" data-saferedirecturl="{{ site_name }}">
+                                {{ site_name }}
+                            </a><wbr>&nbsp;
+
+                            <span style="color:#333333">
+                                {% if full_name%}
+                                    to {{ full_name }}
+                                {% else %}
+                                    to {{ email_address }}
+                                {% endif %}
+                            </span>
+                        </span>
+                    </span>
+                </td>
+            </tr>
+
+            <tr>
+                <td align="center" style="
+                    color: white;
+                    font-size: 12px;
+                    font-family: Arial,Helvetica,sans-serif;">
+                    <div style="
+                        text-align: center;
+                        background: #007FA3;
+                        padding: 15px 0 15px 0;">
+                        Pearson Education Ltd 2018 &nbsp; &nbsp; Registered No: 872828 &nbsp; &nbsp;
+                        <a href="https://www.google.com/maps/place/80+Strand,+London+WC2R+0RL,+UK" target="_blank" data-saferedirecturl="https://www.google.com/url?q=https://www.google.com/maps/place/80%2BStrand,%2BLondon%2BWC2R%2B0RL,%2BUK&amp;source=gmail&amp;ust=1571217499022000&amp;usg=AFQjCNF60-Jsl55ZzfPThyfsZj4phbX1sw" style="
```

Comment:
Please, split the attributes on their own line to avoid line too long. @andrey-canon